frontend/packages/console-shared/src/components/details-page/ExtensionDetailsItem.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { K8sResourceCommon } from '@console/dynamic-plugin-sdk/src/extensions/console-types';
+import { DetailsItem as DetailsItemExtension } from '@console/dynamic-plugin-sdk/src/extensions/details-item';
+import { ResolvedExtension } from '@console/dynamic-plugin-sdk/src/types';
+import { DetailsItem } from '@console/internal/components/utils/details-item';
+
+export const ExtensionDetailsItem: ExtensionDetailsItemComponent = ({ extension, obj }) => {
+  const { path, title, component: Component } = extension.properties;
+  const sortWeight = Number(extension.properties.sortWeight) || Infinity;
+  const content = Component ? (
+    <Component key={extension.properties.id} obj={obj} />
+  ) : (
+    _.get(obj, path, '-').toString()
```

Comment:
When `_.get(obj, path)` returns `null`, this will throw an error. The easiest way is maybe just to use `_.get(obj, path) ?? '-'` ?? But for an object, it would still render "[object Object]".

I don't know if it is too much for here, but maybe we can do something like:

1. typeof value === "string" => just render the string value
2. value === null || value === undefined => "-"
3. else -> use JSON.stringify()? // this should work for booleans, numbers, arrays and objects right?

Wdyt?

Reply:
I think we would only want to render string values if no component is provided. I'll improve the handling of null here.